### PR TITLE
Examples/upload image

### PR DESCRIPTION
**Pull request by devchas**

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java [hunk 1]
```diff
@@ -92,8 +94,8 @@ public static void main(String[] args) throws IOException {
   private void runExample(GoogleAdsClient googleAdsClient, long customerId)
       throws IOException {
 
-    byte[] imageData = com.google.api.ads.common.lib.utils.Media
-      .getMediaDataFromUrl("https://goo.gl/3b9Wfh");
+    byte[] imageData = ByteStreams.toByteArray(new URL(
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java]: nit: Should fit on one line (I think). Max line length is 100.

**Comment by jradcliff** [comment]
@nwbirnie please review when you get a chance.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+    MediaFile file =
+      MediaFile.newBuilder()
+        .setName(StringValue.of("Ad Image"))
+        .setType(MediaType.IMAGE)
+        .setSourceUrl(StringValue.of("https://goo.gl/3b9Wfh"))
+        .setImage(image)
+        .build();
+
+    List<MediaFileOperation> operations = new ArrayList<>();
```
Comment (nwbirnie) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java]: Can you replace this with:

MediaFileOperation op = ...

mediaFileServiceClient.mutateMediaFiles(Long.toString(customerId), Arrays.asList(op))

please. A couple of reasons for this, primarily that it's consistent with our existing examples, and also that new ArrayList() will allocate an array of size 32 to hold on operation.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId)
+      throws IOException {
+
```
Comment (nwbirnie) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadImage.java]: Nit: remove extra whitespace.

**Pull request merged**